CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [v4.10.0](https://github.com/algolia/algoliasearch-client-javascript/compare/4.9.3...4.10.0)
+
+* feat(recommend): introduce Recommend API client (#1280) ([97ebde6](https://github.com/algolia/algoliasearch-client-javascript/commit/97ebde6)), closes [#1280](https://github.com/algolia/algoliasearch-client-javascript/issues/1280) [#1278](https://github.com/algolia/algoliasearch-client-javascript/issues/1278)
```

Comment:
We should maybe mention the deprecation in the changelog, no? https://github.com/algolia/algoliasearch-client-javascript/pull/1278

Reply:
See a1011cf5b9f82b5f64320165dc4299785c7bc846.